src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.cs

```diff
@@ -280,15 +280,12 @@ public bool IsComputingUnusedParams(ISymbol symbol)
                 return false;
             }
 
-            switch (unusedParametersPreference)
+            if (unusedParametersPreference == UnusedParametersPreference.NonPublicMethods)
```

Comment:
should we have some part of our stack look for the value '2' and attempt to change it to '1'?  i.e. in the code that reads/writes this from the VS option system.